monitors/everyone.js

```diff
@@ -0,0 +1,32 @@
+const { Monitor } = require("klasa");
+
+module.exports = class extends Monitor {
+
+  constructor(...args) {
+    super(...args, { ignoreOthers: false });
+  }
+
+  async run(msg) {
+    if (!msg.member || !msg.guild.configs.everyoneRole || !msg.mentions.roles.size || !await msg.hasAtLeastPermissionLevel(1)) return;
+    const everyone = msg.guild.roles.get(msg.guild.configs.everyoneRole);
+    if (!everyone) {
+      await msg.guild.configs.reset("everyoneRole");
+      return;
+    }
+    if (!msg.mentions.roles.has(everyone.id)) return;
+    await msg.member.roles.add(everyone);
```

Comment:
Should check if the bot has manage_roles perms and the everyone role position isn't higher then the author's highest role position